packages/dataviews/src/dropdown-menu-helper.js

```diff
@@ -28,7 +28,10 @@ const radioCheck = (
  * component, which allows deselecting selected values.
  */
 export const DropdownMenuRadioItemCustom = forwardRef(
-	( { checked, name, value, onChange, onClick, ...props }, ref ) => {
+	function DropdownMenuRadioItemCustom(
```

Comment:
React devtools will show `DropdownMenuRadioItemCustom` if we pass a named function to `forwardRef`.